### ISSUE TITLE
Missing separator (newline) when prepending content in changelog file

### ISSUE THREAD

**Issue by alex-fournier** [release-it/conventional-changelog#46]
Hello !

Since [v4.2.0](https://github.com/release-it/conventional-changelog/releases/tag/4.2.0), there is no more newline between the newly prepended changelog and the previous changelog, resulting to something like this:

```
* last line ([1a2b3c](https://gitlab.com/.../commit/1a2b3c))## [1.4.0](https://gitlab.com/.../compare/1.3.1...1.4.0) (2022-04-05)
```

Here is my plugins config in `.release-it.json`:

```json
{
  "plugins": {
    "@release-it/conventional-changelog": {
      "ignoreRecommendedBump": true,
      "infile": "CHANGELOG.md",
      "preset": {
        "name": "conventionalcommits",
        "issueUrlFormat": "https://gitlab.com/{{id}}",
        "types": [
          { "type": "feat", "section": "Features" },
          { "type": "fix", "section": "Bug Fixes" },
          { "type": "chore", "section": "Chore" },
          { "type": "build", "hidden": true },
          {}
        ]
      }
    }
  }
}
```

**Comment by thet** [comment]
same here

**Comment by thet** [comment]
If you want to stay on 4.2.0, this could be a quickfix:

.release-it.js:
```
const fs = require("fs");
const path = require("path");

const main_template = fs
    .readFileSync(
        path.resolve(__dirname, "header-template.hbs")
    )
    .toString();

module.exports = {
    npm: {
        publish: true,
    },
    git: {
        requireBranch: "master",
        commitMessage: "Release new version.",
        commitArgs: ["-n"],
    },
    plugins: {
        "@release-it/conventional-changelog": {
            infile: "CHANGES.md",
            ignoreRecommendedBump: true,
            preset: {
                name: "conventionalcommits",
                types: [
                    {
                        type: "breaking",
                        section: "Breaking Changes",
                    },
                    {
                        type: "feat",
                        section: "Features",
                    },
                    {
                        type: "fix",
                        section: "Bug Fixes",
                    },
                    {
                        type: "maint",
                        section: "Maintenance",
                    },
                ],
            },
            writerOpts: {
                mainTemplate: main_template,
            },
        },
    },
};
```
header-template.hbs:
```
{{> header}}

{{#if noteGroups}}
{{#each noteGroups}}

### ⚠ {{title}}

{{#each notes}}
* {{#if commit.scope}}**{{commit.scope}}:** {{/if}}{{text}}
{{/each}}
{{/each}}
{{/if}}
{{#each commitGroups}}

{{#if title}}
### {{title}}

{{/if}}
{{#each commits}}
{{> commit root=@root}}
{{/each}}

{{/each}}

#
```

This does intend the previous changelog header entry at one level, which is currently acceptable for me.

**Comment by webpro** [comment]
Hopefully it's fixed in [v4.3.0](https://github.com/release-it/conventional-changelog/releases/tag/4.3.0), can you please confirm

**Comment by thet** [comment]
@webpro I can confirm, it works!
Thanks for the fix.
